client/lib/billing/app/views/overlays/billing.js

```diff
@@ -0,0 +1,12 @@
+import OverlayView from 'tahi/views/overlay';
+
+export default OverlayView.extend({
+  templateName: 'overlays/billing',
+  layoutName:   'layouts/overlay',
+  cardName: 'billing',
+
+  didInsertElement: function() {
+    var choice = $(".payment-method .select2-container").select2("val");
+    this.get("controller").set("selectedPaymentMethod", choice);
```

Comment:
actually, ya, it is persisting data, using the Question model.

This snippet is used to set the `selectedPaymentMethod` on page load.
This value is also set/updated onChange of the main select dropdown in the UI, which is used in determining which of the 5 options to show to the user.  

Without this change, the correct options only show after onChange, which doesn't show the option on page load.